osquery/dispatcher/scheduler.h

```diff
@@ -42,6 +45,8 @@ class SchedulerRunner : public InternalRunnable {
 
   /// Maximum number of steps.
   unsigned long int timeout_;
+
+  std::chrono::milliseconds sum_overtime_ = std::chrono::milliseconds::zero();
```

Comment:
I don't see the use for this now so I would not put it in.

Reply:
I have used it for the tests to evaluate the time debt. In the future we can use it to monitoring the sum drift. What do you think?

Reply:
Ok, keep it then. A nit would be to call this drift or time_drift instead of overtime, to me sounds more natural. But just a nit.